tests/schemas/nested_with_float.avsc

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Could you try formatting it like the other schemas ?

Reply:
Is it looks good?

Reply:
perfect !